docs/solidity-by-example.rst

```diff
@@ -110,7 +110,7 @@ of votes.
         }
 
         /// Delegate your vote to the voter `to`.
-        function delegate(address to) public {
+        function delegate(address to, uint proposalChoice) public {
```

Comment:
The param **proposalChoice** conveys the original interest of the sender, towards which the voting rights should be used. **The delegate 'must vote' for this proposalChoice**.